From b4840ed75fcdfa5b7a46437d7210160f6f696a54 Mon Sep 17 00:00:00 2001
From: Zhengqiao Xia <xiazhengqiao@huaqin.corp-partner.google.com>
Date: Tue, 17 Dec 2024 18:52:47 +0800
Subject: [PATCH] CHROMIUM: config: mediatek: Enable typec ucsi

Enable typec ucsi driver

BUG=b:383500182
TEST=emerge kernel and check ui

Change-Id: Idc559df3ca25fa7017707686a546bcd8c3c09e5e
---
 .../config/chromeos/arm64/chromiumos-mediatek.flavour.config     | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chromeos/config/chromeos/arm64/chromiumos-mediatek.flavour.config b/chromeos/config/chromeos/arm64/chromiumos-mediatek.flavour.config
index ab619c1b2278..ab55130ea5df 100644
--- a/chromeos/config/chromeos/arm64/chromiumos-mediatek.flavour.config
+++ b/chromeos/config/chromeos/arm64/chromiumos-mediatek.flavour.config
@@ -254,6 +254,7 @@ CONFIG_TEE=y
 CONFIG_TMPFS=y
 CONFIG_TOUCHSCREEN_MELFAS_MIP4=y
 CONFIG_TYPEC=y
+CONFIG_TYPEC_UCSI=y
 CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_XHCI_MTK=y
 CONFIG_VIDEO_DW9768=m
-- 
2.47.1.613.gc27f4b7a9f-goog

